server/server.go

```diff
@@ -21,6 +21,9 @@ type Server struct {
 	portManger
 }
 
+const customPortForTests = 1337
+const defaultPortForTests = 80
```

Comment:
it would be better that const relate to test moved to xxx_test.go 🙂

Reply:
good suggestion, moved here : https://github.com/status-im/status-go/pull/3172/commits/6839f458b4b3a71c3d5bf7e61e3d80c44131d4fd